data/tekton-pipelines/okd/windows-customize-configmaps.yaml

```diff
@@ -40,7 +40,8 @@ data:
     </unattend>
   sqlserver-install.ps1: |
     #https://github.com/kubevirt/user-guide/pull/645
-    netsh interface ipv4 set subinterface "Ethernet Instance 0" mtu=1200 store=persistent
+    $subinterfaceName = (netsh interface ipv4 show subinterface | Select-String -Pattern "Ethernet").ToString().trim().split() | Select-Object -Last 1
```

Comment:
Should be easier to just do:

`Set-NetIPInterface -InterfaceIndex $(Get-NetAdapter).ifIndex -NlMtuBytes 1200`